raylib_premake5.lua

```diff
@@ -17,6 +17,10 @@ function platform_defines()
     filter {"system:macosx"}
         disablewarnings {"deprecated-declarations"}
 
+    filter {"system:linux"}
+        defines {"_GLFW_X11"}
+        defines {"_GNU_SOURCE"}
+        
```

Comment:
This change does not look required for the UVs.

Reply:
Okay that's my bad.  It doesn't compile on Linux in it's current form and I saw that this issue has been raised on a few other repos so I made the change but you're right, I should have made it a separate pull request.